anni-provider/src/providers/file.rs

```diff
@@ -128,51 +149,29 @@ impl AnniProvider for FileBackend {
     ) -> Result<AudioResourceReader, ProviderError> {
         let path = self.get_disc(album_id, disc_id)?;
         let mut dir = read_dir(path).await?;
-        while let Some(entry) = dir.next_entry().await? {
```

Comment:
`has_album` should also be implemented for `FileBackend`

Reply:
> `has_album` should also be implemented for `FileBackend`

```
/// Returns whether given album exists
    async fn has_album(&self, album_id: &str) -> bool {
        self.albums().await.unwrap_or(HashSet::new()).contains(album_id)
    }
```
There is a default implementation.